contrib/init/50_config

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -eu
+
+: ${CRON_STRING:=0 * * *}
+
+echo 'PATH=/opt/conda/bin:$PATH' > /etc/crontab
+echo "${CRON_STRING} harvester /opt/conda/bin/sensorml2iso -s http://sdf.ndbc.noaa.gov/sos/server.php --verbose --response_formats='text/csv,text/xml;schema=\"ioos/0.6.1\"'  --output_dir=/srv/iso/ndbc 2>&1 | /usr/bin/logger -t sensorml2iso" >> /etc/crontab
+echo "${CRON_STRING} harvester /opt/conda/bin/sensorml2iso -s http://opendap.co-ops.nos.noaa.gov/ioos-dif-sos/SOS --verbose --response_formats='text/csv,text/xml;subtype=\"om/1.0.0/profiles/ioos_sos/1.0\"' --output_dir=/srv/iso/co-ops 2>&1 | /usr/bin/logger -t sensorml2iso" >> /etc/crontab
```

Comment:
This crontab configuration would be more in place in a specifically derived docker image from what should be a generic base.  The old approach using `ENTRYPOINT` yields a very flexible usage scenario: using `sensorml2iso` without having to "install" it in the traditional sense.